scripts/gitlab/cargo-check.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e # fail on any error
+set -u # treat unset variables as error
+echo "________Running cargo_check.sh________"
+
+PARAMS="--target $CARGO_TARGET --locked"
+
+echo "________Validate build________"
+time cargo check $PARAMS --no-default-features
```

Comment:
is this now duplicate to https://github.com/paritytech/parity-ethereum/pull/10452/files#diff-96edf7a6f008de9e928d04e1ae5e12a5R48?

Reply:
yes, forgot to remove the file, thx

Reply:
deleted this file